.github/workflows/_publish-github-release.yml

```diff
@@ -0,0 +1,255 @@
+name: _publish-github-release
+
+on:
+  workflow_call:
+    inputs:
+      release_name:
+        description: 'Name prefix of the release to publish (e.g. "Password Manager")'
+        type: string
+        default: ""
+      workflow_name:
+        description: 'Name of the workflow to check for previous runs (e.g. publish-github-release.yml)'
+        type: string
+        required: true
+      credentials_filename:
+        description: 'Name of the credentials file to download from Azure Blob Storage (e.g. "google-play-credentials.json")'
+        type: string
+        required: true
+      check_release_command:
+        description: >
+          Shell command to check if a release is already published.
+          Use $CREDENTIALS_PATH for the path to credentials file.
+          Example: 'bundle exec fastlane android getLatestVersion serviceCredentialsFile:$CREDENTIALS_PATH'
+        type: string
+        required: true
+
+
+
+jobs:
+  publish-release:
+    name: Publish GitHub Release ${{ inputs.release_name }}
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+      actions: read
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Get latest draft name
+        id: get_latest_draft
```

Comment:
1. 🎨 Other steps are running even when this is empty - which I believe you removed for testing purposes as I'm sure you used to have that. Instead of adding `if: ...` lines back to every single step, we could move this to it's own job?
2.  [Followup](https://github.com/bitwarden/android/pull/5219/files#r2122013536), we can remove the need for multiple workflows by handling a list of draft releases instead, parsing the App from the Release Title.